### PR TITLE
RSS Feed Podcasts

### PR DESCRIPTION
**Pull request by ipatate**

### CODE REVIEW

**Comment by ipatate** [comment]
TODO PR : 
[ ] generate excerpt
[ ] slug

**Comment by ipatate** [comment]
[ ] Last task, generate summary or use new field description

File: content/podcasts/test-podcast-dir/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+publicationDate: YYYY-MM-DD
+status: published
+author: Double Slash
+categories:
+  - Technology
+duration: SSSS
+slug: DS_0XX_text
+videoURL: AjY6K2WFTD0
```
Comment (xlanex6) [inline content/podcasts/test-podcast-dir/index.md:9]: @ipatate 
we publish only youtube id here

File: content/podcasts/test-podcast-dir/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+episodeNumber: XX
+episodeType: full
+explicit: false
+season: 1
+title: full-string-title
+subtitle: full-string-subtitle
+guid: a281e6f01ec88c6c24916751a843e0e9_auto_gen
+links:
```
Comment (xlanex6) [inline content/podcasts/test-podcast-dir/index.md:17]: what about links in front matter ?

**Review by xlanex6 — COMMENTED** [review]
simple suggestion

File: content/podcasts/test-podcast-dir/index.md (hunk 1 above)
Comment (ipatate) [reply content/podcasts/test-podcast-dir/index.md:9]: yes, we can find video with id in RSS. I prefer videoID.

File: content/podcasts/test-podcast-dir/index.md (hunk 2 above)
Comment (ipatate) [reply content/podcasts/test-podcast-dir/index.md:17]: Maybe, we can use a separate file like transcript for only links ?

Comment (ipatate) [reply content/podcasts/test-podcast-dir/index.md]: this format don't work in frontmatter : 
```
- [Azure static Web Apps](https://azure.microsoft.com/fr-fr/services/app-service/static/?WT.mc_id=javascript-00000-wachegha)
- [Documentation Azure static Web Apps](https://docs.microsoft.com/fr-fr/azure/static-web-apps/overview?WT.mc_id=javascript-00000-wachegha)
- [Documentation serverless](https://docs.microsoft.com/fr-fr/azure/azure-functions/create-first-function-vs-code-node?WT.mc_id=javascript-00000-wachegha)
...
```
it's required to write like this : 
```
links: [ {label: Azure static Web Apps, link: https://azure.microsoft.com/fr-fr/services/app-service/static/?WT.mc_id=javascript-00000-wachegha},
{label: Documentation Azure static Web Apps, link: https://docs.microsoft.com/fr-fr/azure/static-web-apps/overview?WT.mc_id=javascript-00000-wachegha},
{label:Documentation serverless, link: https://docs.microsoft.com/fr-fr/azure/azure-functions/create-first-function-vs-code-node?WT.mc_id=javascript-00000-wachegha},...]
```
or use a another file for links.

Comment (xlanex6) [reply content/podcasts/test-podcast-dir/index.md]: I think it's still better to keep all links into the same file episode 
simple 
even if it's not really `nice` on MD file , it do the job.

**Comment by ipatate** [comment]
Done !

**Comment by ipatate** [comment]
RSS generator is done. Don't worries about numbers of files modified. It's just because, i update all the old episodes files.

**Comment by xlanex6** [comment]
All good for me !

**Pull request merged**